packages/flutter/test/material/color_scheme_test.dart

```diff
@@ -252,31 +252,31 @@ void main() {
     expect(scheme.primary, const Color(0xff36618e));
     expect(scheme.onPrimary, const Color(0xffffffff));
     expect(scheme.primaryContainer, const Color(0xffd1e4ff));
-    expect(scheme.onPrimaryContainer, const Color(0xff001d36));
+    expect(scheme.onPrimaryContainer, const Color(0xff194975));
```

Comment:
Re: how-to craft migration guidance 

- Let folks know that the scheme generated from `ColorScheme.fromSeed`, `ColorScheme.fromImageProvider`, and `ThemeData(colorSchemeSeed:..) ` changed based on 0.11.1 --> 0.13.0 of the package
- what properties in the ColorScheme class are affected -  onPrimaryContainer and the others in this PR
- and to manually set those properties to their desired color after generating if they do not like the updated results from the algorithm.

Reply:
Sounds good! Sorry. I wasn't sure what was sufficient guidance given that there is no straight-forward A->B color swap. Thanks for your patience explaining!